scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -1520,19 +1520,23 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
         }
       case KwTry() =>
         next()
+        val startToken = token
+        def getRestIfNeeded(x: Term) =
```

Comment:
It's easier to review when the type is explicitly annotated
```suggestion
        def getRestIfNeeded(x: Term): Term =
```

Reply:
revised